packages/api/cms-api/src/translation/azure-ai-translator.config.ts

```diff
@@ -0,0 +1,5 @@
+export interface AzureAITranslatorConfig {
```

Comment:
Please change this to AzureA**i** in all files to adhere to our internal coding guidelines